packages/account/src/Components/account-limits/account-limits-table-header.tsx

```diff
@@ -1,9 +1,17 @@
-import PropTypes from 'prop-types';
 import * as React from 'react';
 import classNames from 'classnames';
 import { Text } from '@deriv/components';
 
-const AccountLimitsTableHeader = ({ align, children, renderExtraInfo }) => {
+type TAccountLimitsTableHeader = {
+    align?: 'left' | 'right';
+    renderExtraInfo?: () => React.ReactNode;
+};
```

Comment:
when we use Partial, we don’t need ? in props. Partial means, that all props are optional